packages/semcom-node/lib/store/services/component-pod-store.service.ts

```diff
@@ -0,0 +1,51 @@
+import { ComponentMetadata } from '@digita-ai/semcom-core';
+import { Parser, Quad } from 'n3';
+import { ComponentTransformerService } from '../../component/services/component-transformer.service';
+import { ComponentStore } from './component-store.service';
+
+export class ComponentPodStore extends ComponentStore {
+
+  constructor(
+    private readonly uri: string,
+    private readonly transformer: ComponentTransformerService
+  ) { super(); }
+
+  private async fetchAllQuads(): Promise<Quad[]> {
+
+    const body = await fetch(this.uri, { headers: { 'Accept': 'text/turtle' } })
+      .then((response) => response.text());
+
+    const components = new Parser({ format: 'Turtle' }).parse(body)
+      .filter((q) => q.object.value === 'http://www.w3.org/ns/ldp#Resource')
+      .filter((q) => q.subject.value !== '')
+      .map((q) => q.subject.value);
+
+    const allQuads: Quad[][] = await Promise.all(([ ... components ]).map((component) => this.fetchQuads(component)));
```

Comment:
This already was an array...
```suggestion
    const allQuads: Quad[][] = await Promise.all(components.map((component) => this.fetchQuads(component)));
```